tsqa/test_cases.py

```diff
@@ -117,8 +118,12 @@ def tearDownClass(cls):
         # call parent destructor
         super(EnvironmentCase, cls).tearDownClass()
         # if the test was successful, tear down the env
-        if cls.__successful:
-            cls.environment.destroy()  # this will tear down any processes that we started
+        if hasattr(plugin.conf_plugin, 'args'):
```

Comment:
This if logic can be simplified, since this is really just checking "if successful and not cli option"